<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomePage</class>
 <widget class="QWidget" name="HomePage">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1910</width>
    <height>1067</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>150</y>
     <width>1900</width>
     <height>911</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>50</y>
       <width>579</width>
       <height>109</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2"/>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1910</width>
     <height>161</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: lightgrey;</string>
   </property>
   <widget class="QLabel" name="userLabel">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>30</y>
      <width>421</width>
      <height>109</height>
     </rect>
    </property>
    <property name="text">
     <string>尊敬的旅客，欢迎选择SSE！</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>231</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    border-image: url(:/new/prefix1/images/logo.jpg);
    border: 2px solid lightgrey;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>160</y>
     <width>231</width>
     <height>911</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: lightblue;</string>
   </property>
   <widget class="QPushButton" name="searchFlightButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>231</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    font-family: &quot;Microsoft YaHei&quot;;
    font-size: 28px;
    color: #2AC4D9;
background-color:white;


}
QPushButton:hover {
background-color:#FAC85C;
}</string>
    </property>
    <property name="text">
     <string>查询航班</string>
    </property>
   </widget>
   <widget class="QPushButton" name="myOrdersButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>231</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    font-family: &quot;Microsoft YaHei&quot;;
    font-size: 28px;
    color: #2AC4D9;
background-color:white;

}
QPushButton:hover {
background-color:#FAC85C;
}</string>
    </property>
    <property name="text">
     <string>我的订单</string>
    </property>
   </widget>
   <widget class="QPushButton" name="seatsSelectButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>231</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    font-family: &quot;Microsoft YaHei&quot;;
    font-size: 28px;
    color: #2AC4D9;
background-color:white;

}
QPushButton:hover {
background-color:#FAC85C;
}</string>
    </property>
    <property name="text">
     <string>值机选座</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>210</y>
      <width>231</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    font-family: &quot;Microsoft YaHei&quot;;
    font-size: 28px;
    color: #2AC4D9;
background-color:white;

}
QPushButton:hover {
background-color:#FAC85C;
}</string>
    </property>
    <property name="text">
     <string>个人中心</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>280</y>
      <width>231</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    font-family: &quot;Microsoft YaHei&quot;;
    font-size: 28px;
    color: #2AC4D9;
background-color:white;

}
QPushButton:hover {
background-color:#FAC85C;
}</string>
    </property>
    <property name="text">
     <string>信箱</string>
    </property>
   </widget>
  </widget>
  <zorder>widget</zorder>
  <zorder>stackedWidget</zorder>
  <zorder>widget_3</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
